frontend/src/store/Post/postAction.ts

```diff
@@ -61,9 +61,9 @@ export const searchAction = (
 ) => {
   return (dispatch: Redux.Dispatch<PostDispatchType>) => {
     return axios
-      .post<SearchType[]>(`/post/search/`, searchForm)
+      .post<{ordinary: SearchType[], like: SearchType[], created: SearchType[]}>(`/post/search/`, searchForm)
       .then((res) => {
-        dispatch({ type: SEARCH_SUCCESS, payload: res.data});
+        dispatch({ type: SEARCH_SUCCESS, ordinary: res.data.ordinary, like: res.data.like, date: res.data.created});
```

Comment:
Why did you use a different name from backend? (data <-> created) If you don't have any other reason, why don't you make them same?